### PR TITLE
Reuse existing roll for critical hits

### PR DESCRIPTION
**Pull request by BoltsJ**
When creating critical hit rolls, instead of rolling an all new damage roll,
take the existing roll and smoosh it into a second new roll, then keep the
appropriate number of dice.

Apparently there's no existing issue for this one.

### CODE REVIEW

File: src/module/macros.ts [hunk 1]
```diff
@@ -396,6 +395,7 @@ export async function renderMacroTemplate(actor: LancerActor | undefined, templa
     terms: [],
     result: "",
   };
+  /*
```
Comment (SohumB) [inline src/module/macros.ts:398]: I'd rather just delete the code; it'll be in git history if we need it

File: src/module/helpers/actor.ts [hunk 2]
```diff
@@ -130,8 +131,8 @@ export function clicker_stat_card(
 ): string {
   let button = "";
   // Determine whether this is an unlinked token, so we can encode the correct id for the macro.
-  const r_actor = options.data.root.actor;
-  let id = r_actor.token && !r_actor.token.isLinked ? r_actor.token.id : r_actor.actor.id!;
+  const r_actor = options.data.root.actor as LancerActor;
+  let id = r_actor.token && !r_actor.token.isLinked ? r_actor.token.id : r_actor.id!;
```
Comment (SohumB) [inline src/module/helpers/actor.ts:135]: I assume this is for an unrelated bugfix?

**Review by SohumB — APPROVED** [review]

File: src/module/macros.ts (hunk 1 above)
Comment (BoltsJ) [reply src/module/macros.ts]: Whoops, I meant to. Fixed now

File: src/module/helpers/actor.ts (hunk 2 above)
Comment (BoltsJ) [reply src/module/helpers/actor.ts:135]: It was breaking linked sheets for me, so I had fix this just to get it tested.

**Pull request merged**